cumulus/test/service/src/lib.rs

```diff
@@ -507,8 +507,11 @@ where
 					block_import_handle: slot_based_handle,
 					spawner: task_manager.spawn_handle(),
 				};
-
-				slot_based::run::<Block, AuthorityPair, _, _, _, _, _, _, _, _, _>(params);
+				let params_with_export = SlotBasedParamsWithExport {
+					params,
+					None
```

Comment:
This does not compile because you need to provide a name for this struct field. But check the other comment, this should just be `Params`.